frontend/src/modules/admin/components/AdminHeader.vue

```diff
@@ -171,22 +171,6 @@ const logoutAdmin = async () => {
               }}</span>
             </div>
             <ul class="py-1 text-nearWhite" aria-labelledby="dropdown">
-              <li>
```

Comment:
whyd you delete this ? put it back Keep it here just make router push to my profile component . i.e i click on my profile from that dropdown i navige to the myprofile like i do i click myprofile in the nav bar

Reply:
nice